tools/docker/daml-all-in-one/healthcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# docker exec -it eb40b692a662 /bin/bash
+
+curl -X GET http://localhost:7575/v1/query   -H "Content-Type: application/json"   -H "Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwczovL2RhbWwuY29tL2xlZGdlci1hcGkiOnsibGVkZ2VySWQiOiJzYW5kYm94IiwiYXBwbGljYXRpb25JZCI6ImZvb2JhciIsImFjdEFzIjpbIkFsaWNlIl19fQ.1Y9BBFH5uVz1Nhfmx12G_ECJVcMncwm-XLaWM40EHbY"
```

Comment:
@raynatopedrajeta  ^^ Plus one on that review point and I'll chime in with my own:
The bearer token hardcoded in the healthcheck script is going to cause breakages when we upgrade the image later (I'm guessing) so is there any chance that the shell script could look up the bearer token at runtime instead? I'm assuming it's either stored in a file somewhere on the file-system of the container or otherwise accessible somehow programmatically).

If none of this seems possible easily then please provide a detailed explanation of how you've obtained the bearer token hardcoded in the sources right now and then we can work our way backwards from there.

Reply:
Hello @petermetz ,

I am thinking of storing the auth bearer token via .env file. What are your thoughts?

Cheers!

Reply:
Hello @outSH ,

This issue has been addressed. Thanks!

Reply:
> Hello @petermetz ,
> 
> I am thinking of storing the auth bearer token via .env file. What are your thoughts?
> 
> Cheers!

@raynatopedrajeta I'd look into generating it at runtime, not at build time. As long as that's what you meant I'm OK with the .env file being created at runtime and then the healthcheck script reading that .env file.

Reply:
Hello @petermetz ,
I've created the .env file that will store the generated auth bearer token.

FYI: this token has been manually generated at[jwt.io]( https://jwt.io/) following the instructions on [DAML api docs](https://docs.daml.com/json-api/index.html#using-claim-tokens)

Cheers!

Rayn

Reply:
> Hello @petermetz , I've created the .env file that will store the generated auth bearer token.
> 
> FYI: this token has been manually generated at [jwt.io](https://jwt.io/) following the instructions on [DAML api docs](https://docs.daml.com/json-api/index.html#using-claim-tokens)
> 
> Cheers!
> 
> Rayn

@raynatopedrajeta OK, in that case: I'd look into generating it at runtime, not at build time.